blog-highlights/demystifying-appflowy-editors-codebase.md

```diff
@@ -755,16 +752,16 @@ We can make a selection within our Editor using the `updateSelection()` method.
 await editor.updateSelection(Selection.single(path: [0], startOffset: 0));
 ```
 
-To access nodes within your editor. You can do so with:
+To access nodes within your editor, you can use the `nodeAtPath` method as shown below:
 ```dart
 final node = editor.nodeAtPath([2]);
 ```
 
-Here we are accessing the node located at path [2].
+Here we are accessing the node located at path [2]. *TODO: maybe a explain this a bit better -- are we simply saying we are choosing the 3rd node in the collection of nodes?*
```

Comment:
Yes, a better replacement for this will be: 

Here we are choosing the 3rd node in the editor.